src/Ozds.Client/Shared/Print/NetworkUserCalculationPrintAbbreviations.razor

```diff
@@ -1,26 +1,26 @@
 <p>
-  <u>Značenje kratica:</u>
+  <u>@T["The meaning of acronyms"]:</u>
 </p>
-<p style="margin-top: 0.8rem;"><strong>JEN</strong> – prekomjerno preuzeta jalova energija</p>
-<p><strong>IND</strong> – induktivna jalova energija</p>
-<p><strong>KAP</strong> – kapacitivna jalova energija</p>
-<p><strong>MJT</strong> – mrežarina u doba u doba jedinstvene dnevne tarife</p>
-<p><strong>MNT</strong> – mrežarina u doba u doba niže dnevne tarife</p>
-<p><strong>MU</strong> – naknada za mjernu uslugu</p>
-<p><strong>MVT</strong> – mrežarina u doba više dnevne tarife</p>
-<p><strong>O</strong> - očitano</p>
+<p style="margin-top: 0.8rem;"><strong>@T["JEN"]</strong> – @T["excessively drawn reactive power"]</p>
+<p><strong>@T["IND"]</strong> – @T["inductive reactive power"]</p>
+<p><strong>@T["KAP"]</strong> – @T["capacitive reactive power"]</p>
+<p><strong>@T["MJT"]</strong> – @T["grid fee during the period of a uniform daily tariff"]</p>
+<p><strong>@T["MNT"]</strong> – @T["grid fee during the period of a lower daily tariff"]</p>
+<p><strong>@T["MU"]</strong> – @T["measurement service fee"]</p>
+<p><strong>@T["MVT"]</strong> – @T["grid fee during the period of a higher daily tariff"]</p>
+<p><strong>@T["O"]</strong> - @T["read "]</p>
 <p>
-  <strong>OIEiK</strong> – naknada za obnovljive izvore energije i
-  visokoučinkovitu kogeneraciju
+  <strong>@T["OIEiK"]</strong> – @T["fee for renewable energy sources and high-efficiency cogeneration"]
 </p>
-<p><strong>OMM</strong> – obračunsko mjerno mjesto</p>
-<p><strong>P</strong> - procjena</p>
-<p><strong>RJT</strong> – radna energija u doba u doba jedinstvene dnevne tarife</p>
+<p><strong>@T["OMM"]</strong> – @T["billing metering point"]</p>
+<p><strong>@T["P"]</strong> - @T["estimate"]</p>
```

Comment:
this is my fault for leaving it here - we have no estimates right no so please change this line to
```html
<p><strong>@T["O"]</strong> - @T["read "]</p>
```